<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生理期信息记录系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QCalendarWidget" name="caleCurr">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edCurr">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listLevel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNoteDown">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>记录当天</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除当天</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>401</width>
     <height>611</height>
    </rect>
   </property>
   <property name="title">
    <string>当天数据</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>体重（kg）：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>流量：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>疼痛感觉：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfoPaintPic">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfoWeight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #004695;
font: 75 18pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfoUser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>261</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetFile">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0072C6;
    color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font-size: 16px;
    padding: 6px;
}
QPushButton:hover {
    background-color: #148CD2;
}
QPushButton:pressed {
    background-color: #005CA3;
}</string>
    </property>
    <property name="text">
     <string>更换头像</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbFlow">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	font:9pt;
	border-radius:5px;
	text-align:center;
	border:1px solid #E8EDF2;
	background-color: rgb(255, 255, 255);
	border-color: rgb(180, 180, 180);
}
QProgressBar:chunk{
	border-radius:5px;
	background-color:#1ABC9C;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生理期记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoosePain">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0072C6;
    color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font-size: 16px;
    padding: 6px;
}
QPushButton:hover {
    background-color: #148CD2;
}
QPushButton:pressed {
    background-color: #005CA3;
}</string>
   </property>
   <property name="text">
    <string>疼痛等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当天流量：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>预计持续时间(天)：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edSpend">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入天数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当天体重（kg）：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edWeight">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入体重</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPaintPic">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>400</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sliderFlow">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>450</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QSlider {
        border-radius: 10px;
    }
    QSlider::groove:horizontal {
        height: 5px;
        background: #FFFFFF
    }
    QSlider::handle:horizontal{
        background: #0072C6;
        width: 16px;
        height: 16px;
        margin:-6px 0;
        border-radius:8px;
    }
    QSlider::sub-page:horizontal{
        background:#f90;
    }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbFlow">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
